src/components/FormikField/FormikField.stories.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
```

Comment:
I think the docs doesn't render as expected. For instance:
- when pressing `Show code`, the entire code is presented, but I assume we only want the bit from `render` to be displayed.
- the default value of component prop in props table is not listed in DEFAULT column.

The first issue also appears in `Card` stories, which is one of the few other `.stories.tsx` docs. There are also some slight inconsistencies in presentation between `.stories.tsx` and `.stories.mdx` docs e.g. the Props header missing in `.stories.tsx` docs.

Reply:
> I think the docs doesn't render as expected. For instance:
> 
> * when pressing `Show code`, the entire code is presented, but I assume we only want the bit from `render` to be displayed.

There's an open issue here, with a workaround that I've applied to both files: https://github.com/storybookjs/storybook/issues/22281.

> * the default value of component prop in props table is not listed in DEFAULT column.

It looks like the table couldn't handle the `Input` component as a default but this can be set via a jsdoc string so I've done that.

> 
> The first issue also appears in `Card` stories, which is one of the few other `.stories.tsx` docs. There are also some slight inconsistencies in presentation between `.stories.tsx` and `.stories.mdx` docs e.g. the Props header missing in `.stories.tsx` docs.

It looks like it was decided not to customise the page too much and stick with the CSF defaults: https://github.com/canonical/react-components/pull/1010#discussion_r1447061301.

Reply:
Thanks for the explanation and links, and for also fixing the inconsistency in `Card` stories!